src/components/FeatureViews/Registration/RegistrationCart/PaymentBoard.js

```diff
@@ -260,9 +345,9 @@ export default function PaymentBoard() {
 		history.push(`/registration/receipt/${payment.data.createPayment.payment.id}`)
 	};
 
-	if (error) {
-		console.error(error.message);
-		return <div>There has been an error! : {error.message} </div>
+	if (error || enrollmentResponse.error) {
+		console.error(error.message, enrollmentResponse.error.message);
+		return <div>There has been an error! : {error.message} | {enrollmentResponse.error.message}</div>
```

Comment:
Did you mean to write `{error.message || enrollmentResponse.error.message}` instead? As it is, if one errors but not the other, there will be an extra `|`.

Reply:
i was only thinking about the situation if there are 2 errors. I think of some better way of handling this.